dev/PushNotifications/PushNotificationManager.cpp

```diff
@@ -245,4 +245,36 @@ namespace winrt::Microsoft::Windows::PushNotifications::implementation
             LOG_IF_FAILED(::CoRevokeClassObject(static_cast<DWORD>(token.Cookie())));
         }
     }
+
+    bool PushNotificationManager::IsBIAvailable()
+    {
+        try
+        {
+            BackgroundTaskBuilder builder = BackgroundTaskBuilder();
+            auto builder5 = builder.as<winrt::IBackgroundTaskBuilder5>();
```

Comment:
NIT: [ES.23: Prefer the {}-initializer syntax](https://isocpp.github.io/CppCoreGuidelines/CppCoreGuidelines.html#es23-prefer-the--initializer-syntax)

```
auto builder{ BackgroundTaskBuilder() };
auto builder5{ builder.as<...>(); };
```

Same comment applies throughout